### PR TITLE
refactor: log

### PR DESCRIPTION
**Pull request by alili**

### CODE REVIEW

File: src/views/dashboard/data-explorer/components/log.vue [hunk 1]
```diff
@@ -2,25 +2,25 @@
 a-list(:hoverable="true" size="small")
   template(#header)
     | {{$t('dataExplorer.logs')}}
-  a-list-item(v-for="item of logListData" :key="item")
-    a-tooltip(:content="item.error ? item.error : item.runCode")
+  a-list-item(v-for="item of logs" :key="item")
+    a-tooltip(:content="item.error ? item.error : item.sql")
```
Comment (ZonaHex) [inline src/views/dashboard/data-explorer/components/log.vue:6]: Are we sure we want to change 'code' to 'sql'? What if our editor can support other languages such as python in the future?

File: src/store/modules/log/index.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+  getters: {},
+
+  actions: {
+    pushLog(log: any) {
+      this.logs.push(log)
+    },
+    clearLogs() {
+      this.logs = []
+    },
```
Comment (ZonaHex) [inline src/store/modules/log/index.ts:18]:
```suggestion
```
Could use '$reset()' from pinia.

File: src/store/modules/code-run/index.ts [hunk 3]
```diff
@@ -22,65 +22,48 @@ const useCodeRunStore = defineStore('codeRun', {
...
+          ...res.output[0],
+        })
+      } catch (error: any) {
+        pushLog({
+          sql,
+          ...error,
+        })
       }
     },
+    // todo: can we combine next two logs into one line code?
```
Comment (ZonaHex) [inline src/store/modules/code-run/index.ts:66]:
```suggestion
```

File: src/store/modules/log/index.ts [hunk 4]
```diff
@@ -0,0 +1,21 @@
+import { getSqlResult } from '@/api/editor'
+import { Message } from '@arco-design/web-vue'
```
Comment (ZonaHex) [inline src/store/modules/log/index.ts:2]:
```suggestion
```

File: src/views/dashboard/data-explorer/components/log.vue (hunk 1 above)
Comment (alili) [reply src/views/dashboard/data-explorer/components/log.vue:6]: > 

I think pyhon has a lot of lines of code and should display logs in a different component than SQL requests

**Review by ZonaHex — APPROVED** [review]

**Pull request merged**